tests/test_axis_limits.py

```diff
@@ -272,8 +276,11 @@ def test_limit_continuity(self):
         # The need for a weaker tolerance on these keys may be due to a subpar
         # polynomial regression fit against which the axis limit is compared.
         weaker_tolerance = {
-            "iota_r": {"atol": 1e-6},
-            "iota_num_rr": {"atol": 5e-5},
+            "B0_rr": {"rtol": 5e-03},
+            "iota_r": {"atol": 1e-4},
+            "iota_num_rr": {"atol": 5e-3},
+            "alpha_r": {"rtol": 1e-3},
+            "cvdrift0": {"atol": 1e-5},
```

Comment:
The PR doesn't include anything that would require reducing these tolerances. Do you get errors otherwise?

Reply:
Done.